docs/fsharp-cheatsheet.md

```diff
@@ -349,15 +349,16 @@ A *statically resolved type parameter* is a type parameter that is replaced with
 
 ####
 
-    type TypeA = { Id: string; A: int }
-    type TypeB = { Id: string; B: int }
+    type RequestA = { Id: string; StringValue: string }
+    type RequestB = { Id: string; IntValue: int }
 
-    let inline getId<'t when 't : (member Id: string)> (x: 't) = x.Id
-    let a : TypeA = { Id = "A"; A = 11 }
-    let b : TypeB = { Id = "B"; B = 12 }
+    let requestA : RequestA = { Id = "A"; StringValue = "Value" }
+    let requestB : RequestB = { Id = "B"; IntValue = 42 }
 
-    let idOfTypeA = getId a
-    let idOfTypeB = getId b
+    let inline getIdOfRequest<'t when 't : (member Id: string)> (x: 't) = x.Id
```

Comment:
also, cant recall foibles of the syntax, but can/should the member spec be on its own line?

Reply:
I'll change back to `getId`. I agree, I should have left it. Got overzealous.

I found this on the member spec: https://learn.microsoft.com/en-us/dotnet/fsharp/style-guide/formatting#formatting-explicit-generic-type-arguments-and-constraints
One line is fine if it's short enough, otherwise it gets split.

Reply:
can/should the html edition link there? pdf too?